### DISCUSSION TITLE
Failed to process line error after starting the game

### DISCUSSION THREAD

**Discussion by Biokax** [ppy/osu#18426] [Q&A] [answered]
Just after starting the game I got this error messages
![image](https://user-images.githubusercontent.com/50368878/170547689-cb2e3a3b-e77f-4eb2-af19-0866efb90d4f.png)
I think they came from the unranked maps I downloaded recently
Logs :
[updater.log](https://github.com/ppy/osu/files/8781085/updater.log)
[runtime.log](https://github.com/ppy/osu/files/8781084/runtime.log)
[database.log](https://github.com/ppy/osu/files/8781081/database.log)
[performance.log](https://github.com/ppy/osu/files/8781083/performance.log)
[network.log](https://github.com/ppy/osu/files/8781082/network.log)

`
2022-05-26 17:54:25 [verbose]: ⚠️ Failed to process line "244,201,32455,6,0,B|869:40|869:40|504:-523|-281:-499|-505:131|-139:695|-139:695|149:1142|814:1187|1160:487|869:40|869:40|1211:-181|1211:-181|623:-1087|-426:-722|-1062:7|-473:912|-473:912|32:1694|963:1417|1719:599|1211:-181|1211:-18…
2022-05-26 17:54:25 [verbose]: ⚠️ Subsequent messages have been logged. Click to view log files.
`

**Comment by bdach** [comment, marked as answer]
these sorts of error messages come from maps that exploit undefined behaviour in stable, by which i mean things like: objects with too large/too small coordinates, infinite bpm control points and similar.

see also: https://github.com/ppy/osu/issues/750

**Reply by smoogipoo** [reply]
Part of me says we should probably silence them from notifications though and/or figure out some other way to display it (like maybe a warning icon when this is the case).

**Reply by Biokax** [reply]
> Part of me says we should probably silence them from notifications though and/or figure out some other way to display it (like maybe a warning icon when this is the case).

I think this error is for the mapper, not for the player, because the player can't do anything to fix that error from a downloaded beatmap except delete it, and the message is to big to make it understandable without opening the logs

**Reply by peppy** [reply]
Resolved via #18435.
